tests/zfs-tests/cmd/xattrtest/xattrtest.c

```diff
@@ -168,6 +168,7 @@ parse_args(int argc, char **argv)
 			break;
 		case 'p':
 			strncpy(path, optarg, PATH_MAX);
+			path[PATH_MAX - 1] = '\0';
```

Comment:
@behlendorf 
to update this issue. with new pr: https://github.com/zfsonlinux/zfs/pull/5296
please review it.
thanks.